db/migrate/20241104082851_create_annual_report_statuses_per_account_counts.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateAnnualReportStatusesPerAccountCounts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :annual_report_statuses_per_account_counts do |t| # rubocop:disable Rails/CreateTableWithTimestamps
```

Comment:
Obviously we can do this with a future migration, but I wonder if naming-wise something like `AnnualAccountStat` (to essentially mirror the existing `AccountStat` which acts as a value cache, but this one on the yearly level) would make sense as a home for future values related to annual reporting or otherwise.